index.bs

```diff
@@ -1049,6 +1074,203 @@ Note: Invocations of {{MLContext/compute()}} will fail if any of the {{MLContext
+          }),
+          context.createTensor({
+            dataType: 'float32', shape: [2, 2], readable: true
+          })
+        ]);
+
+    // 4. Initialize the inputs.
+    context.writeTensor(inputTensorA, new Float32Array(4).fill(1.0));
+    context.writeTensor(inputTensorB, new Float32Array(4).fill(0.8));
+
+    // 5. Execute the graph.
+    const inputs = {
+      'A': inputTensorA,
+      'B': inputTensorB
+    };
+    const outputs = {
+      'C': outputTensorC
+    };
+    context.dispatch(graph, inputs, outputs);
+    
+    // 6. Read back the computed result.
+    const result = await context.readTensor(outputTensorC);
+    console.log('Output value:', new Float32Array(result));  // [1, 1, 1, 1]
+  </pre>
+</details>
+</div>
+
+### {{MLContext/createTensor()}}  ### {#api-mlcontext-createtensor}
+
+Creates an {{MLTensor}} associated with this {{MLContext}}.
+
+<div dfn-for="MLContext/createTensor(descriptor)" dfn-type=argument>
+    **Arguments:**
+      - <dfn>descriptor</dfn>: an {{MLTensorDescriptor}}.
+
+    **Returns:** {{Promise}}<{{MLTensor}}>.
+</div>
+
+<details open algorithm>
+  <summary>
+    The <dfn method for=MLContext>createTensor(|descriptor|)</dfn> method steps are:
+  </summary>
+    1. Let |global| be [=this=]'s [=relevant global object=].
+    1. Let |tensor| be the result of [=creating an MLTensor=] given [=this=], and |descriptor|.
+    1. Let |promise| be [=a new promise=].
+    1. Enqueue the following steps to [=this=]'s {{MLContext/[[timeline]]}}:
+        1. Create |tensor|'s {{MLTensor/[[data]]}} given |descriptor| and initialize all bytes to zeros.
+        1. If that fails, then [=queue an ML task=] with |global| to [=reject=] |promise| with an "{{UnknownError}}" {{DOMException}}, and abort these steps.
+        1. Otherwise, [=queue an ML task=] with |global| to [=resolve=] |promise| with |tensor|.
+    1. Return |promise|.
+</details>
+
+### {{MLContext/readTensor(tensor)}}  ### {#api-mlcontext-readtensor}
+
+Reads back the {{MLTensor/[[data]]}} of an {{MLTensor}} from the {{MLContext}}'s {{MLContext/[[timeline]]}} to script.
+
+<div dfn-for="MLContext/readTensor(tensor)" dfn-type=argument>
+    **Arguments:**
+      - <dfn>tensor</dfn>: an {{MLTensor}}. The tensor to be read.
+
+    **Returns:** {{Promise}}<{{ArrayBuffer}}>. A buffer containing the result of the read.
+</div>
+
+<details open algorithm>
+  <summary>
+    The <dfn method for=MLContext>readTensor(|tensor|)</dfn> method steps are:
+  </summary>
+    1. Let |global| be [=this=]'s [=relevant global object=].
+    1. Let |realm| be [=this=]'s [=relevant realm=].
+    1. If |tensor|'s {{MLGraph/[[context]]}} is not [=this=], then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+    1. If |tensor|'s {{MLTensor/[[isDestroyed]]}} is true, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+    1. If |tensor|'s {{MLTensor/readable}} is false, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
```

Comment:
The definition for MLTensor/readable has minor issues (see below) but I think this should read "If tensor is readable ..." or "If tensor.[[descriptor]].["readable"] ..."

Reply:
This is now "If |tensor|.{{MLTensor/[[descriptor]]}}["{{MLTensorDescriptor/readable}}"] is false..."

Reply:
Oops, sorry... I forgot we don't use map syntax for dictionaries [we use dotted property syntax for those too](https://github.com/webmachinelearning/webnn/blob/main/docs/SpecCodingConventions.md#dictionary-members), so it should be:

"If tensor.[[descriptor]].readable ..."